samples/Azure.Management.Storage/Generated/Extensions/SubscriptionExtensions.cs

```diff
@@ -38,7 +38,7 @@ public static AsyncPageable<SkuInformation> ListSkuInformationsAsync(this Subscr
                 var restOperations = GetSkusRestOperations(clientDiagnostics, credential, options, pipeline, subscription.Id.SubscriptionId, baseUri);
                 async Task<Page<SkuInformation>> FirstPageFunc(int? pageSizeHint)
                 {
-                    using var scope = clientDiagnostics.CreateScope("SubscriptionExtensions.ListSkuInformations");
+                    using var scope = clientDiagnostics.CreateScope("SubscriptionExtensions.ListSkus");
```

Comment:
2 issues:
method name should be `ListSkuInformationAsync`
scope should be `SubscriptionExtensions.ListSkuInformation`

Reply:
This is resolved.